src/applications/mhv/secure-messaging/components/shared/SmBreadcrumbs.jsx

```diff
@@ -130,13 +171,13 @@ const SmBreadcrumbs = () => {
               if (crumb.path.includes('https://')) {
                 return (
                   <a key={i} href={crumb.path}>
-                    Return to {crumb.label}
+                    Back to {crumbs[crumbs.length - 2]?.label.toLowerCase()}

```

Comment:
what happens here if `crumbs[crumbs.length - 2]` is not set ooc?

Reply:
it wont bring up the crumb since it will trigger the condition in line 176: `crumbs.length > 1 && (`